### PR TITLE
feat: polish Chapter Card layout

### PR DESCRIPTION
**Pull request by Sboonny**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

A cleaner follow-up to #1711, without the noises related to other changes I have added

Important points https://github.com/freeCodeCamp/chapter/pull/1711#issuecomment-1282450095 and https://github.com/freeCodeCamp/chapter/pull/1711#issuecomment-1282450095

### CODE REVIEW

**Comment by Sboonny** [comment]
IDK, which is better

image

image

**Comment by Sboonny** [comment]
Maybe it's better to swap about section and new events section :thinking:

**Comment by Sboonny** [comment]
Moving the filter function into its own PR, like how event are filtered in home page

File: client/src/components/ChapterCard.tsx [hunk 1]
```diff
@@ -1,106 +1,131 @@
...
+          fontWeight={'500'}
+          paddingInline={'1em'}
+          marginBlockStart={'.5em'}
+          gridArea="eventheader"
+        >
+          New Events
+        </Heading>
+        <GridItem area="event" paddingInline={'1em'}>
+          {chapter.events.map(({ id, name, start_at }) => (
+            <>
```
Comment (ojeytonwilliams) [inline client/src/components/ChapterCard.tsx:110]:
```suggestion
```
There's only one child

Comment (ojeytonwilliams) [reply client/src/components/ChapterCard.tsx:110]: This will mess up the formatting, sorry!

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
It's looking much nicer, @Sboonny!

Just some nitpicks:

File: client/src/components/ChapterCard.tsx [hunk 2]
```diff
@@ -1,106 +1,131 @@
...
+              <Link key={id} href={`/events/${id}`} _hover={{}}>
```
[the remark this answers is not in the document]
Comment (Sboonny) [reply client/src/components/ChapterCard.tsx:111]: Yeab, that reminds me, I should make a consistent style for links 😵‍💫

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**